controlplane/kubeadm/internal/etcd/fake/client.go

```diff
@@ -18,173 +18,41 @@ package fake
 
```

Comment:
This file was out of date since it was unused entirely. I updated it so I could fake etcd responses to simulate the situations necessary for the test.